modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/data/handler/BatchEnginePortletDataHandlerUtil.java

```diff
@@ -53,6 +53,13 @@ public static Map<String, Serializable> buildExportParameters(
 				() -> {
 					List<String> batchNestedFields = new ArrayList<>();
 
+					if (MapUtil.getBoolean(
+							portletDataContext.getParameterMap(),
+							PortletDataHandlerKeys.COMMENTS)) {
+
+						batchNestedFields.add("comments");
+					}
+
 					batchNestedFields.add("customFields.attributeType");
```

Comment:
@guilhermedcamacho  maybe this should be moved to the top of this new conditional block that you added?